app/views/rooms/show.html.haml

```diff
@@ -0,0 +1,10 @@
+.container
+  %h2= @room.name
+  .room-description= @room.description
+  .room-capacity= "#{@room.reviewees.count} / #{@room.capacity}"
+  .room-reviewer= @room.reviewer.name
+  - if current_user.participatable?(@room)
+    %p= link_to "ルームに参加する", participation_path(@room), method: :put
+  - if current_user.participating_rooms.exists?(@room.id)
+    %p= link_to "ルームから退出する", participation_path(@room), method: :delete, data: { confirm: "ルームから退出してよろしいですか？" }
```

Comment:
__[HAML-Lint]__

Line is too long. [111/80]


<!-- Comment by [SideCI](https://sideci.com) -->